app/src/main/java/com/aaronjwood/portauthority/network/Wireless.java

```diff
@@ -186,6 +186,38 @@ public int getInternalWifiSubnet() {
         return netmask;
     }
 
+
+    public int getInternalWifiIpAddressAsInt() {
```

Comment:
If you want you can take the other method that returns a string and make it generic so that we have one method for both (have a generic return type based on some class you pass it).